src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -69,13 +71,13 @@ def __init__(self, task_request):
             self._TRANSFORM_TIMEOUT = rospy.get_param('~transform_timeout')
             self._MAX_HORIZ_SPEED = rospy.get_param('~max_translation_speed')
             self._MAX_START_TASK_DIST = rospy.get_param('~roomba_max_start_task_dist')
+            if not self._check_max_roomba_range():
```

Comment:
When I was talking about the valueerror I was saying we need to check that the parameters roomba_max_start_task_dist and the magnitude of x_overshoot and y_overshoot are congruent.

I don't think this function can be called yet because self._roomba_point is None